mycpp/mylib_old.h

```diff
@@ -58,55 +61,17 @@ void print(Str* s);
 // log() generates code that writes this
 void println_stderr(Str* s);
 
-class IndexError {};
-class ValueError {};
-class KeyError {};
-
-class EOFError {};
-
-class NotImplementedError {
- public:
-  NotImplementedError() {
-  }
-  explicit NotImplementedError(int i) {  // e.g. in expr_to_ast
-  }
-  explicit NotImplementedError(const char* s) {
-  }
-  explicit NotImplementedError(Str* s) {
-  }
-};
-
-class AssertionError {
- public:
-  AssertionError() {
-  }
-  explicit AssertionError(int i) {  // e.g. in expr_to_ast
-  }
-  explicit AssertionError(const char* s) {
-  }
-  explicit AssertionError(Str* s) {
-  }
-};
-
-// Python's RuntimeError looks like this.  . libc::regex_match and other
-// bindings raise it.
-class RuntimeError {
- public:
-  RuntimeError(Str* message) : message(message) {
-  }
-  Str* message;
-};
-
 //
 // Data Types
 //
 
-#ifdef MYLIB_LEAKY
+int len(Str *s);
+
 class Str : public gc_heap::Obj {
  public:
   Str(const char* data, int len)
       : gc_heap::Obj(Tag::FixedSize, gc_heap::kZeroMask, 0),
-        len_(len),
+        len__(len),
```

Comment:
Hm this looked weird but I guess it's to expose all instances as compile errors, OK

Reply:
Correct.  I didn't bother changing it back because (hopefully) we'll be deleting this Soon™